ExecutorPluginNG-ERP-4.0/SimulationPlugin.cs

```diff
@@ -0,0 +1,38 @@
+﻿using DistributedExperimentation.DataModel;
+using DistributedExperimentation.Experimenter.ExecutorPlugin;
+using System;
+using System.Collections.Generic;
+
+namespace ExecutorPluginNGERP40
+{
+    public class ExecutorPlugin : IExecutorPlugin
+    {
+        public void execute(IExperimentSeries experimentSeries)
+        {
+            var experimentCollection = experimentSeries.getExperiments();
+            foreach (var experiment in experimentCollection)
```

Comment:
I get an Experiment Collection back but shouldnt i get an Single Eperiment instance ?

Reply:
The Experimenter-Software expected an ExperimentSeries with only one Experiment inside. If contains the given ExperimentSeries various Experiment's than take the first Experiment easily. But this case should not occur until the Investigator briefed the Experimenter automatically. Normal, it has always only one Experiment inside.

The Experimenter receives higher-level information about the series of experiments by passing the ExperimentSeries additionally. I think, this can improved by an more strict json schema that allowed only one Experiment pro ExperimentSeries but there is currently no need.

Reply:
If you keep it that way, Id prefer to have an globalParmaters: []  in the Json file and a GetGlobalParameters() in the ExperimentSeries.